llvm/include/llvm/IR/IntrinsicsSPIRV.td

```diff
@@ -62,4 +62,5 @@ let TargetPrefix = "spv" in {
   def int_spv_any : DefaultAttrsIntrinsic<[llvm_i1_ty], [llvm_any_ty]>;
   def int_spv_lerp : Intrinsic<[LLVMMatchType<0>], [llvm_anyfloat_ty, LLVMMatchType<0>,LLVMMatchType<0>], 
     [IntrNoMem, IntrWillReturn] >;
+  def int_spv_rsqrt : DefaultAttrsIntrinsic<[LLVMMatchType<0>], [llvm_anyfloat_ty]>;
```

Comment:
We need a test for this.  Can you add a spirv target to https://github.com/llvm/llvm-project/blob/117921e071a353edbd27f08456ec27ea98ecdb8c/clang/test/CodeGenHLSL/builtins/rsqrt.hlsl